beacon-chain/core/state/spectest/slot_processing_test.go

```diff
@@ -63,6 +64,25 @@ func runSlotProcessingTests(t *testing.T, config string) {
 				t.Log(diff)
 				t.Fatal("Post state does not match expected")
 			}
+
+			// Process slots and epoch with optimizations.
+			f := featureconfig.Get()
+			f.OptimizeProcessEpoch = true
+			featureconfig.Init(f)
+			if c := featureconfig.Get(); !c.OptimizeProcessEpoch {
+				t.Errorf("OptimizeProcessEpoch in FeatureFlags incorrect. Wanted true, got false")
+			}
+
+			postState, err = state.ProcessSlots(context.Background(), beaconStateCopy, beaconStateCopy.Slot+uint64(slotsCount))
+			if err != nil {
+				t.Fatal(err)
+			}
+
+			if !proto.Equal(postState, postBeaconState) {
+				diff, _ := messagediff.PrettyDiff(beaconState, postBeaconState)
+				t.Log(diff)
```

Comment:
Maybe put this in fatal ?

Reply:
Also why are we processing slots twice for this test ?

Reply:
It's fatal below: `t.Fatal("Post state does not match expected")`
Twice because:
1.) process slots without process epoch optimization 
2.) process slots with process epoch optimization (Feature flag enabled at line 70)

Reply:
ah no I meant something like 
```
t.Fatalf("Post state does not match expected. Diff between states %s",diff)
```

Reply:
Ah i see. Updated. Thanks!